templates/zerver/help/pin-a-stream.md

```diff
@@ -0,0 +1,43 @@
+# Pinning a stream
+
+Pinning a stream will put that stream at the top of your streams sidebar. If you are subscribed to a lot of streams, pinning your favorite streams can be a good way to keep things organized.
```

Comment:
Can you wrap the markdown so each line is < 76 characters?  This won't affect rendering (good), but it will make look nicer in most tools (also good).